koku/reporting/migrations/0095_auto_20200212_1606.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.2.10 on 2020-02-12 16:06
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0094_auto_20200211_1449")]
+
+    operations = [
+        migrations.RunSQL(
+            """
+            DELETE FROM reporting_awscostentrylineitem
+            WHERE cost_entry_bill_id IS NULL;
+
+            DELETE FROM reporting_awscostentrylineitem_daily
+            WHERE cost_entry_bill_id IS NULL;
+
+            DELETE FROM reporting_awscostentrylineitem_daily_summary
+            WHERE cost_entry_bill_id IS NULL;
+
+            DELETE FROM reporting_azurecostentrylineitem_daily
+            WHERE cost_entry_bill_id IS NULL;
+
+            DELETE FROM reporting_azurecostentrylineitem_daily_summary
+            WHERE cost_entry_bill_id IS NULL;
+
+            DELETE FROM reporting_ocpusagelineitem
+            WHERE report_period_id IS NULL;
+
+            DELETE FROM reporting_ocpusagelineitem_daily
+            WHERE report_period_id IS NULL;
+
+            DELETE FROM reporting_ocpusagelineitem_daily_summary
+            WHERE report_period_id IS NULL;
+
+            DELETE FROM reporting_ocpstoragelineitem
+            WHERE report_period_id IS NULL;
+
+            DELETE FROM reporting_ocpstoragelineitem_daily
+            WHERE report_period_id IS NULL;
+            """
```

Comment:
Here is a list of other reporting tables that have report_period_id. Do these tables need to be considered as well?
```
                      table_name                      |   column_name    
------------------------------------------------------+------------------
 reporting_ocpawscostlineitem_daily_summary           | report_period_id
 reporting_ocpawscostlineitem_project_daily_summary   | report_period_id
 reporting_ocpazurecostlineitem_daily_summary         | report_period_id
 reporting_ocpazurecostlineitem_project_daily_summary | report_period_id
 reporting_ocpcosts_summary                           | report_period_id
 reporting_ocpusagereport                             | report_period_id
```

Reply:
Will update.

Reply:
The only other question that I have is: if this data is not supposed to be in the system, should it be blocked with a NOT NULL constraint on that FK?

Reply:
Capturing this in a follow-up PR. We should do the NOT NULLs, but it also breaks MANY tests, and this is blocking QE now, so we should get it out as the first step.